code/obj/item/clothing/suits.dm

```diff
@@ -2067,6 +2067,15 @@ TYPEINFO(/obj/item/clothing/suit/space/industrial/salvager)
 		src.AddComponent(/datum/component/wearertargeting/energy_shield, list(SLOT_WEAR_SUIT), 0.8, 1, FALSE, 2)
 		src.AddComponent(/datum/component/power_cell, 100, 100, 5, 30 SECONDS, TRUE)
 
+/obj/item/clothing/suit/security_badge/paper
+	name = "Hall Monitor Badge"
+	desc = "A piece of soggy notebook paper with a red S doodled on it, presumably to represent security."
+	icon = 'icons/obj/clothing/overcoats/item_suit_gimmick.dmi'
+
+	wear_image_icon = 'icons/mob/clothing/overcoats/worn_suit_gimmick.dmi'
+	inhand_image_icon = 'icons/mob/inhand/overcoat/hand_suit_gimmick.dmi'
```

Comment:
You don't need to put icon files here as they are the same as parent suit/security_badge